.github/workflows/publish.yml

```diff
@@ -0,0 +1,33 @@
+# This will run every time a tag is created and pushed to the repository.
+# It calls our tests workflow via a `workflow_call`, and if tests pass
+# then it triggers our upload to PyPI for a new release.
+name: Publish to PyPI
+on:
+  release:
+    types: ["published"]
+
+jobs:
+  tests:
+    uses: ./.github/workflows/tests.yml
+  publish:
+    name: publish
+    needs: [tests] # require tests to pass before deploy runs
```

Comment:
Do we also need 'docs-audit' here, or 'tests' is good enough you think?

Reply:
This will run _all_ of the jobs that are in that workflow file, so it'll run `docs-audit` as well 👍